councilmatic_core/management/commands/import_data.py

```diff
@@ -355,12 +351,12 @@ def grab_organizations(self):
     def grab_organization_posts(self, org_dict):
         url = base_url + '/organizations/'
 
-        r = session.get(url, params=org_dict)
+        r = session.get(url, params=org_dict, timeout=60)
         page_json = json.loads(r.text)
         organization_ocd_id = page_json['results'][0]['id']
 
         url = base_url + '/' + organization_ocd_id + '/'
-        r = session.get(url)
+        r = session.get(url, timeout=60)
```

Comment:
OK, Not sure what order these comments will render in but, you get the idea. Or you will.